lib/targets/truffle/generation.ts

```diff
@@ -123,6 +124,7 @@ function generateInputType(evmType: EvmType): string {
     case AddressType:
       return "string | BigNumber";
     case BytesType:
+    case DynamicBytesType:
```

Comment:
Commenting separately from my other suggestion (to add binary types to this mapping), because this looks to be wrong. It's fine to use a `BigNumber` for the `BytesType`, since it's mapped from the `bytes1 .. bytes32` Solidity types, which all fit in a single EVM word (so also a single `BigNumber`). On the other hand, `bytes` can have arbitary length, so it's wrong to use a single `BigNumber` for this.